### DISCUSSION TITLE
Chatting about sansio_lsp_client

### DISCUSSION THREAD

**Discussion by Moosems** [tomlin7/biscuit#338] [General]
@tomlin7 I noticed you made a PR to sansio_lsp_client and I'm hoping to learn how to use it for a personal project. Would you be able to help me make some docs for it so that its easier to learn and get started with? I'd love to discuss this outside of biscuit on something like Discord or somewhere else so as to not clog up biscuit discussions but if you'd prefer it here that works too. Hope you're having a good day!

**Comment by tomlin7** [comment]
Sure lets have chats on discord, that would be more easier

I'm in a tkinter server, rdbende is present so you can have a chat with one of the contributors of the lib
https://discord.com/invite/6AW9z6hw

Also do invite me to the servers with tkinter devs if possible, I'd love to talk to people :)
